### ISSUE TITLE
Fix landmark for phase banner

### ISSUE THREAD

**Issue by edhamiltonHO** [HO-CTO/engineering-guidance-and-standards#130]

**Comment by keithkennedyHO** [comment]
Closing due to comments in #132 

> we will not fail the build based on negative aXe result relating to breadcrumbs or phase banner being outside landmark region
